addons/html_editor/static/src/core/zws_plugin.js

```diff
@@ -38,9 +38,28 @@ export class ZwsPlugin extends Plugin {
             case "CLEAN":
                 this.clean(payload.root);
                 break;
+            case "NORMALIZE":
+                this.normalize(closestElement(payload.node));
```

Comment:
I think payload.node is guaranteed to be an element (or null), so `closestElement` is not needed here.
See History Plugin's `addStep` and `getMutationsRoot`.